dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -259,25 +253,42 @@ describe("AppViewComponent", () => {
       expect(err.html()).toContain("Application mr-sunshine not found");
     });
 
+    it("renders the loading icon if the URLs table is not ready", () => {
+      const wrapper = mount(<AppViewComponent {...validProps} />);
+      const ingress = {
+        isFetching: true,
+      };
+      const ingresses = { foo: ingress };
+      wrapper.setState({ ingresses });
+
+      expect(wrapper.find(AccessURLTable)).not.toExist();
+      const loading = wrapper.find(LoadingWrapper);
```

Comment:
Maybe you can reuse `itBehavesLike(aLoadingComponent)`

https://github.com/kubeapps/kubeapps/blob/33d6bf4a47c3e4a3e95ab9a59d55cdb6bf8371fb/dashboard/src/components/UpgradeForm/UpgradeForm.test.tsx#L14

Reply:
`itBehavesLike("aLoadingComponent", ` checks if there is a LoadingWrapper within the given component. We cannot use that here because:

 - If we pass `AppView` as the component it may find LoadingWrapper not related to the component we are interested in. For example it can detect a LoadingWrapper around the ServiceTable but we are interested in the AccessURLTable.
 - If we pass the component that we want (AccessURLTable for example) it won't work because the LoadingWrapper is actually outside the component.

Reply:
After some refactoring I have moved the different LoadingWrappers so I can use now the `itBehavesLike` method.